docker/docker-compose.yml

```diff
@@ -6,6 +6,10 @@ services:
       context: .
     ports:
       - "8787:8787"
+    volumes:
+      - type: bind
+        source: ../
+        target: /lakes
```

Comment:
This is just to make it more convenient to use RStudio in the container locally. Good for testing certain things if HPC is too busy.